grails-app/controllers/org/pih/warehouse/inventory/StockMovementController.groovy

```diff
@@ -193,7 +192,7 @@ class StockMovementController {
         def stockMovements
 
         try {
-            stockMovements = stockMovementService.getStockMovements(stockMovement, [max: max, offset: offset, createdAfter: createdAfter, createdBefore: createdBefore])
+            stockMovements = stockMovementService.getStockMovements(stockMovement, params)
```

Comment:
Doesn't look like createdAfter and createdBefore are added back to the params map above.